site/faq/index.md

```diff
@@ -64,9 +64,9 @@ StarlingX is a top-level pilot project supported by the OpenStack Foundation, bu
 
 The StarlingX team is actively working with a number of other projects and communities, both within and outside of the OpenStack Foundation projects. The community is contributing to the Nova, Horizon, Keystone and Neutron projects within OpenStack. Our teams are also involved in the OSF Edge Computing Group as well as the Akraino community and the EdgeXFoundry project.
 
-#### When is the first release of StarlingX?
+#### When is the next release of StarlingX?
 
-The first release will be in October 2018. The project will release on a 4 month cadence each March, July and November in 2019. Releases are time based.
+The first release was in October 2018. The next release is in August 2019. 
 
```

Comment:
There's a separate PR to update this particular answer: https://github.com/StarlingXWeb/starlingx-website/pull/41

Either here or in the other PR, I would go for an alternative and say: StarlingX 1.0 was released in October, 2018. You can now download and try out StarlingX 2.0 which was just released at the end of August, 2019. The next release is targeted towards the end of 2019 and the community is planning to keep having two releases per year onwards.

Reply:
We agreed with Yong to make the changes to this question here. We would like both the question and answer to be updated as:

"What is the release cadence for StarlingX?"

"The initial (beta) StarlingX release was released in October 2018. In 2019 and moving forward, StarlingX releases will align with OpenStack releases. StarlingX will have two releases a year, following a cycle-trailing model relative to the OpenStack release cycle."

Reply:
This one is complete.